ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -94,17 +94,6 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-aws-ipi-ovn
-- as: aws-ipi-proxy-sts-f28
```

Comment:
any reason to remove these 3 jobs? is it because we don't have same 4.14 night e2e jobs?

Reply:
if we remove these 3 jobs, we will have fewer OTA case OCP-24358.

Reply:
I moved these jobs to multi-arch ci together with OTA cases. The motive of the migration is to control the upgrade ci jobs in a balance. 

From my last review experience for the PR that you added ci profiles, per the rules of "adding all e2e profiles for y/z upgrade ci in each arch ci", take `aws-ipi-proxy-sts`, if we follow the rule, the e2e job would add 2 upgrade job (one is for z-stream, another is for y-stream) in amd, another 2 in arm. In total, it is 4 jobs. That consumes our resources a lot.